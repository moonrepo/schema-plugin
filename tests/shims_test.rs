use proto_pdk_test_utils::*;
use starbase_sandbox::locate_fixture;

#[cfg(not(windows))]
generate_shims_test!(
    "schema-test",
    [],
    Some(locate_fixture("schemas").join("base.toml"))
);
