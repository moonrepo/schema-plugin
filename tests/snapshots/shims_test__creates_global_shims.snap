---
source: tests/shims_test.rs
expression: "std::fs::read_to_string(sandbox.path().join(\".proto/shims\").join(if cfg!(windows)\n                    {\n                    format!(\"{}.cmd\", \"schema-test\")\n                } else { \"schema-test\".to_string() })).unwrap()"
---
#!/usr/bin/env bash
set -e

if [ -n "$PROTO_DEBUG" ]; then
    set -x
    echo "Running with schema-test.sh shim"
fi

exec proto run schema-test --  "$@" 

